sdk/Pulumi.Automation.Tests/LocalWorkspaceTests.cs

```diff
@@ -736,6 +736,38 @@ public async Task StackLifecycleInlineProgram()
             }
         }
 
+        [Fact]
+        public async Task PreviewDestroy()
+        {
+            var program = PulumiFn.Create(() => { });
+            Assert.IsType<PulumiFnInline>(program);
+
+            var stackName = $"{RandomStackName()}";
+            var projectName = "inline_node";
```

Comment:
inline_dotnet maybe

Reply:
I'd call it `inline_dotnet`, since it's an inline dotnet program.  `inline_node` above is probably just copy pasta from the nodejs automation api tests.